mixins.sass

```diff
@@ -5,3 +5,9 @@
 @mixin item
     background-color: $itembackground
     border-radius: 10px
+
+@mixin mobile
```

Comment:
mobile is too generic, you can specify what would you like to do like:
- item-mobile, box-mobile, etc